### PR TITLE
[46] Product Page using HTML - Mobile Track

### PR DESCRIPTION
**Pull request by Oluwabusiolami**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		Stage 7. The topics and situations in Cambridge Global English have been selected to reflect this diversity and encourage learning 
+		about each other's lives through the medium of English. It presents realistic listening, speaking, reading and writing tasks, as 
+		well as end-of-unit projects similar to those students might encounter in the context of a first-language school. After every 
+		other unit, there is a literature spread, featuring authentic texts from a variety of sources. CEFR Level - B1
+              </p>
+            </div>
+        </div>
+        <hr />
+       <nav>
+        <ul>
```
Comment (TundeOladejo) [inline submissions/oluwabusolamiEluyela/html/index.html:68]: This tag is not used anywhere. Remove it.

Comment (TundeOladejo) [inline submissions/oluwabusolamiEluyela/html/index.html:68]: You are not using this ul tag. Remove it.

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <center><h1><a href="#">JUSTIFY</a></h1></center>
+      </div>
+      <nav>
+        <ul>
+	    <center><h2><a href="#">Hot Deals</a>
+            <a href="#">Gift Cards</a>
+            <a href="#">Sell</a><c/enter>
+            <a href="#">Book Rentals</a></h2></center>
+
+             <li><a href="#">New Releases</a></li><li> <a href="#">General Book Shelf</a></li><li> <a href="#">Today's Deals</a></li><li> <a href="#">Registration</a></li><li> <a href="#">Teachers Corner</a></li><li> <a href="#">Students Corner</a></li><li> <a href="#">Contact</a></li><li> <a href="#">Order Offline</a></li>
```
Comment (TundeOladejo) [inline submissions/oluwabusolamiEluyela/html/index.html:20]: This is not cool. Please ensure your codes are properly indented. And I mean for this whole page too.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+				<h5>$23.30</h5>
+
+ 	  <a href="#">Cambridge Global English Teachers Resource Stage 7</a></li>
+				<h5>$40.50</h5>
+   
+          </li>
+        </ol>
+      </div>
+    </main>
+   <center><a href="#"><footer> Conditions of Use     Privacy Notice    &copy 2019-2020, Justify.com</footer></center>
```
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html/index.html:84]: Place the anchor tag inside the footer tag instead. No need for the center tag. When you start doing CSS, you can centralize your work

**Review by adaugochi — COMMENTED** [review]
Remove spaces on image names. use lower cases also

**Pull request closed without merging**